### PR TITLE
IT-5156 thick scrollbar under s breakpoint

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/styles/design-system.scss [hunk 1]
```diff
@@ -50,7 +50,9 @@ textarea {
 }
 
 html.thick-scrollbar {
-	@include scrollbar(15px, var(--raw-gray-100));
+	@media #{breakpoint-s()} {
```
Comment (tweetgeek) [inline lib/styles/design-system.scss:53]: what about the tablet?

Comment (pioziol123) [reply lib/styles/design-system.scss]: right :)
https://bethinkteam.slack.com/archives/C012R8RM3A4/p1682688253072869

Comment (tweetgeek) [reply lib/styles/design-system.scss]: > Dopytać UX co z tabletem

**Pull request merged**